package.json

```diff
@@ -19,42 +19,58 @@
   "files": [
     "dist"
   ],
-  "main": "dist/server/index.js",
-  "types": "dist/server/index.d.ts",
+  "main": "server.js",
   "scripts": {
-    "build": "bash run build-client && bash run build-server",
-    "lint": "bash ./run lint",
+    "build": "bash run build",
+    "lint": "bash run lint",
     "prepack": "yarn build",
-    "test": "bash ./run update && bash ./run start"
+    "start": "bash run start",
+    "test": "yarn update && yarn start",
+    "update": "bash run update",
+    "watch": "bash run watch"
   },
   "dependencies": {
     "chalk": "^2.4.2",
-    "express": "^4.17.0",
-    "global-tunnel-ng": "^2.7.1",
-    "got": "^9.6.0",
-    "lodash": "4.17.15"
+    "global-agent": "^2.1.5",
+    "got": "^9.6.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-proposal-numeric-separator": "^7.2.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-typescript": "^7.6.0",
     "@n4bb12/config-tslint": "^1.4.1",
     "@types/express": "^4.16.1",
-    "@types/global-tunnel-ng": "^2.1.0",
     "@types/got": "^9.4.1",
     "@types/lodash": "^4.14.123",
     "@types/node": "^12.0.4",
     "@verdaccio/types": "^8.1.0",
```

Comment:
I just released `@verdaccio/types: 8.2.0 => 8.3.0` your PR is included there.

Reply:
sweet!